src/com/palmergames/bukkit/towny/db/TownyFlatFileSource.java

```diff
@@ -658,9 +658,16 @@ public boolean loadTown(Town town) {
 					for (String token : tokens) {
 						if (!token.isEmpty()) {
 							TownyMessaging.sendDebugMsg("Town Fetching Outlaw: " + token);
-							Resident outlaw = getResident(token);
-							if (outlaw != null)
-								town.addOutlaw(outlaw);
+							try
+							{

```

Comment:
Can we get the `{`'s to line up with how the rest of the codebase is? 🙈

Reply:
I agree, I took in the one PR on the /ta resident {resident} unjail and noticed it.

Reply:
I think what you meant to say was let's fix the rest of towny to this obviously superior indentation.